es/index.js

```diff
@@ -0,0 +1,156 @@
+import toDate from './lib/toDate';
```

Comment:
Since we are having to repeat all the code in the `/es` folder, it means changes have to be made in duplicate. Could we think of a build process where this code is generated into the folder and all the necessary transformations made? 🤔

Reply:
This is done already, please see "build:es" script in package.json.

Reply:
Oh, got it 👍